packer/install-dependencies.sh

```diff
@@ -436,3 +436,5 @@ sudo mv /var/log /var/log.vhd
 sudo mkdir /var/log
 
 applyCIS
+cloud-init clean --logs
```

Comment:
this can be done from the packer json

see https://github.com/hashicorp/packer/blob/master/examples/azure/linux_custom_image.json#L39

https://www.packer.io/docs/builders/azure.html#deprovision

Reply:
Is there any benefit to using the packer interface?

Reply:
1) by keeping the deprovision step separated from this script, the install-dependencies script can be reused as-is in other contexts (eg. managed image builder) as it only does what it's meant to, ie. install stuff on the VM
2) we also have a cleanup script which is currently running after install-dependencies. Deprovision should always be the very last step packer runs.
3) the Azure packer docs examples do it that way https://www.packer.io/docs/builders/azure.html#deprovision